exchange/bidder.go

```diff
@@ -98,7 +98,7 @@ const (
 // The name refers to the "Adapter" architecture pattern, and should not be confused with a Prebid "Adapter"
 // (which is being phased out and replaced by Bidder for OpenRTB auctions)
 func AdaptBidder(bidder adapters.Bidder, client *http.Client, cfg *config.Configuration, me metrics.MetricsEngine, name openrtb_ext.BidderName, debugInfo *config.DebugInfo, endpointCompression string) AdaptedBidder {
-	return &bidderAdapter{
+	return &BidderAdapter{
```

Comment:
I am not sure why you started exporting `BidderAdapter` ?

Reply:
Oh, nevermind, I see. You need it to extract the mock adapter out of the exchange.